### ISSUE TITLE
Section 3 2 Credentials makes confusing statements about "credentials"

### ISSUE THREAD

**Issue by mwherman2000** [w3c/vc-data-model#989]
Reference: https://www.w3.org/TR/vc-data-model/#credentials

The opening paragraph includes several references/uses of the word "credential" that muddy the definition of what the "credential" component is with the context of a "verifiable credential".

**Comment by mwherman2000** [comment]
This situation/confusion is made worse by examples such as https://www.w3.org/TR/vc-data-model/#example-usage-of-the-id-property that include a tab named "Credential" that doesn't correspond with the (admittedly difficult) text of Section 3.2.

**Comment by mwherman2000** [comment]
The definition of "credential" in https://www.w3.org/TR/vc-data-model/#terminology is pretty good.

It's imperative that the usage and intended meaning of the word "credential" be made more consistent across the entire spec.

**Comment by David-Chadwick** [comment]
I agree that the definition is pretty good and it clearly separates a credential from a verifiable credential. Therefore both the credential and verifiable credential can have their own metadata (such as validity time, id etc).

**Comment by RieksJ** [comment]
The VCDM states that a VC is-a credential, which I read as "a subtype (or subclass) of". This means that any instance of a VC is also an instance of 'Credential', and not some kind of derived (and therefore _other_) credential, in the same way that "cat" is a subclass of "mammal" and having a mammal that happens to be a cat doesn't mean that there are two distinct things. However, as in many places the VCDM is not as specific as could be, I can see that people feel they can, sometimes even need to, take some liberty in interpreting texts. 

I think the case of @David-Chadwick may also resemble that what you can see in passports: there is a passport number that identifies the passport, and some other identifier (a SSN in the USA, BSN in the Netherlands, etc.) that identifies the person whose data is written in the passport. The passport and the person-related data in the passport would be different things.

**Comment by David-Chadwick** [comment]
@RieksJ thankyou for this observation. The next thing is to convince @jandrieu about this model as in (#978) he does not think the credential ever exists in its own right. But mammals do exist, even ones that are not cats.

**Comment by TallTed** [comment]
> some other identifier (a SSN in the USA, BSN in the Netherlands, etc.) that identifies the person whose data is written in the passport

Semi-tangent — The SSN does not identify any person, as it is not a universally unique number, assigned once to one human and never again. The SSN *plus* the associated human name _do_ comprise a unique value, associated with a single human. There will never be another human who has my complete name as recorded by the Social Security Administration *and* the SSN which was assigned to me -- but there are (for certain) other humans who have my complete name and a different SSN, and there *may* be other humans who have the same SSN and a different name.

This is a semi-tangent because relating our work to, and basing our work on, existing models requires that we fully and completely understand those existing models. This comment among others shows that we don't yet have that full and complete understanding of the SSN, and quite possibly, of other existing models which we're trying to emulate and draw analogies to.

**Comment by Sakurann** [comment]
A PR addressing Issue #1009 would address this one too. To converge discussions, closing this issue.
